### DISCUSSION TITLE
404 Upon Installation with Raspberry Pi 4 + Docker Compose

### DISCUSSION THREAD

**Discussion by kchou0907** [firefly-iii/firefly-iii#6900] [Firefly III]
I'm trying to run the base installation on my Raspberry Pi 4 using Docker Compose. It seems like everything installed successfully, but when I try to navigate to the website using the Pi's ip address, I just get a 404 page not found. There's nothing in the console besides not found either - I'm starting to wonder if I'm overlooking something really simple.

I basically followed [this video](https://www.youtube.com/watch?v=w_wdA60pePc), only diverging because I'm pretty sure you have to make sure the db login matches between the .env and .yml files.

I want to run this on a raspberry pi 4 running on Debian 

My config file:

> version: '3.3'
> 
> services:
>   app:
>     image: fireflyiii/core:latest
>     restart: always
>     volumes:
>       - firefly_iii_upload:/var/www/html/storage/upload
>     env_file: .env
>     ports:
>       - 80:8080
>     depends_on:
>       - db
>   db:
>     image: mariadb    
>     hostname: fireflyiiidb
>     restart: always
>     environment:
>       - MYSQL_RANDOM_ROOT_PASSWORD=yes
>       - MYSQL_USER=kchou0907
>       - MYSQL_PASSWORD=<secret but I swear it matches with what's in the env file>
>       - MYSQL_DATABASE=firefly
>     volumes:
>       - firefly_iii_db:/var/lib/mysql
> volumes:
>    firefly_iii_upload:
>    firefly_iii_db:

**Comment by JC5** [comment]
Is the container running correctly? What do its logs say?

**Reply by kchou0907** [reply]
Here's the full log:

[log.txt](https://github.com/firefly-iii/firefly-iii/files/10471361/log.txt)

**Reply by JC5** [reply]
Mmm, so it works correctly but nothing reaches the container. You did match the ports. What happens when you do something like `curl http://127.0.0.1` on the command line of your RPi?

**Reply by kchou0907** [reply]
Also receiving a 404 error!

**Reply by JC5** [reply]
But nothing in the logs?

**Reply by kchou0907** [reply]
Nope - logs are the same as before

**Reply by kchou0907** [reply]
Oops! Nvm super dumb mistake. I forgot I had previously been experimenting with K3s. Thank you for your help!

**Reply by JC5** [reply]
No problem, glad to hear you found it!
